### PR TITLE
Fix a couple of autoyast errors importing the configuration 

### PR DESCRIPTION
**Pull request by scabrero**
Fixes https://bugzilla.suse.com/show_bug.cgi?id=1185499 and https://bugzilla.opensuse.org/show_bug.cgi?id=1184630.

### CODE REVIEW

**Comment by nodeg** [comment]
Awesome. Thank you for handling this!

**Review by dgdavid — APPROVED** [review]
Looks good, thanks!

For future references, this is the same than #101 (`SLE-15-SP3` branch) and #102 (`SLE-15-SP2` branch) but for `master` branch.

File: src/lib/auth/authconf.rb [hunk 1]
```diff
@@ -475,12 +475,18 @@ def ldap_export
 
         # Set configuration for LDAP from exported objects.
         def ldap_import(exported_conf)
-            @ldap_conf = exported_conf['conf']
-            @ldap_conf = {} if @ldap_conf.nil?
-            @ldap_pam = exported_conf['pam']
-            @ldap_pam = false if @ldap_pam.nil?
-            @ldap_nss = exported_conf['nss']
-            @ldap_nss = [] if @ldap_nss.nil?
+            if exported_conf.nil?
```
Comment (teclator) [inline src/lib/auth/authconf.rb:478]: np: I would use fetch for this initialization, something like:

```ruby
@ldap_conf = (exported_conf || {}).fetch('conf, {})
@ldap_pam = @ldap_conf.fetch('pam', false)
@ldap_nss = @ldap_conf.fetch('nss', [])
```

Comment (teclator) [reply src/lib/auth/authconf.rb:478]: I overlooked @dgdavid suggestions (https://github.com/yast/yast-auth-client/pull/102#discussion_r626572201)

Comment (scabrero) [reply src/lib/auth/authconf.rb:478]: Thanks @teclator and @dgdavid for the suggestions, they are highly appreciated. I am a completely ruby newbie :-)

**Pull request merged**